rad/monadic/issues.rad

```diff
@@ -4,7 +4,24 @@
 ;; Create a remote issue chain. Returns a ref for the chain that you
 ;; can assign to a variable.
 
-(load! "rad/prelude.rad")
+(load! "rad/prelude/test.rad")
```

Comment:
Is it no longer possible to have prelude re-export a bunch of things?

Reply:
No it is, I was just debugging this file and wanted to see exactly what was getting imported. (Because if you run it from normal repl, it would load prelude twice, etc.) I can change it back.